### DISCUSSION TITLE
建议添加直接上传轮播图片的功能

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#425] [游戏或网站运行反馈]
如图所示，只有“轮播图URL”这个区块，所谓的能够上传图片的“轮播图片”区块并不存在。
![图片](https://github.com/Gamuxorg/bbs/assets/53572814/e205bebf-1a45-4037-8176-f8ec31e01942)
![图片](https://github.com/Gamuxorg/bbs/assets/53572814/bc85b42d-9fe6-4cea-ac9c-fd7cbcd62262)

并非每一个游戏都是在Steam上发售、可直接抓取图片的。对于Steam上没有的游戏，想传点自己的游戏截图，没有直接的上传入口，只能曲线救国：例如我最近上传的 [你即操作系统](https://www.linuxgame.cn/youre-the-os)，所有的图片都是是在“特色图片”里上传，再把链接提供给“轮播图URL”的。

这样的图片显然还是能直接上传最方便，希望能够添加这个功能。

**Comment by hyh19962008** [comment]
麻烦提到 gamuX 这个项目下，简单描述成“轮播图区块添加上传图片功能即可”

**Comment by hyh19962008** [comment]
已添加
